document/CoreFormat.mdk

```diff
@@ -2439,9 +2440,35 @@ C++: `"std::runtime_error"`
 C#: `"System.ArgumentNullException"`
 ~
 
+If the tool does not have access to information about the object that was thrown, the `type` property shall be absent.
+
+~ IsoNote
+This will be the case in C++ programs that throw objects that do not derive from `std::exception`:
```

Comment:
I understand that if ex is actually a derived type of runtime_error, then I won't get that. But is this better than nothing? Or too hacky?